nautobot/extras/tests/test_plugins.py

```diff
@@ -272,6 +283,86 @@ def test_registry_nav_menu_dict(self):
         )
 
 
+@skipIf(
+    "dummy_plugin" not in settings.PLUGINS,
+    "dummy_plugin not in settings.PLUGINS",
+)
+class PluginWebhookTest(APITestCase):
```

Comment:
This is pretty much an exact copy of extras.tests.test_webhooks.WebhookTest, right?

I don't know that this test provides a lot of value here - basically it's just testing that if we call `process_webhook` with a canned set of data, it constructs and sends an appropriately formatted HTTP request. The only difference between this test and `WebhookTest` is that you're providing a slightly different set of data (a different Webhook pk, a different serializer data dictionary, a different model name string) - there's nothing here that really makes this a plugin-specific test or really tests anything that isn't already covered by `WebhookTest`.

Can this instead be implemented more like the `extras.tests.test_customfields.CustomFieldBackgroundTasks` test case, with inheriting from `CeleryTestCase` and verifying that making a model change through an HTTP request results in a correctly formed `process_webhook` task being enqueued?